src/models/main.ts

```diff
@@ -37,6 +45,14 @@ export class ContainerTreeItem<T extends IdItem> extends vscode.TreeItem {
   get children(): T[] {
     return this._children;
   }
+
+  searchableText(): string {
+    let label = this.label;
+    if (this.label && typeof this.label !== "string") {
```

Comment:
Can this not be expressed in positive terms about it being a TreeItemLabel and then simplifying the dereference on next line?

Reply:
Unfortunately there isn't any direct way of saying `if (this.label instanceof vscode.TreeItemLabel)` since it's just an interface, so we have to rule out `undefined` and `string` manually